pandas/_libs/tslibs/dtypes.pyx

```diff
@@ -362,6 +359,8 @@ cdef dict c_PERIOD_AND_OFFSET_DEPR_FREQSTR = {
     "MIN": "min",
 }
 
+INVALID_FREQ_ERR_MSG = "Invalid frequency: {0}"
```

Comment:
```suggestion
cdef str INVALID_FREQ_ERR_MSG = "Invalid frequency: {0}"
```

Reply:
done

Reply:
I think CI failures are unrelated to my changes.